internal/jobservice/repository/sqlite.go

```diff
@@ -374,3 +367,35 @@ func (s *JSRepoSQLite) GetSubscribedJobSets(ctx context.Context) ([]SubscribedTu
 	}
 	return tuples, nil
 }
+
+func (s *JSRepoSQLite) purgeExpiredJobsAndJobSets() {
+	ticker := time.NewTicker(time.Duration(s.jobServiceConfig.PurgeJobSetTime) * time.Second)
+	log := log.WithField("JobService", "ExpiredJobSetsPurge")
+	jobsetsStmt := fmt.Sprintf(`DELETE FROM jobsets WHERE Timestamp < (UNIXEPOCH() - %d);`, s.jobServiceConfig.PurgeJobSetTime)
+	jobServiceStmt := fmt.Sprintf(`DELETE FROM jobservice WHERE Timestamp < (UNIXEPOCH() - %d);`, s.jobServiceConfig.PurgeJobSetTime)
+	tableExistsStmt := `SELECT EXISTS (SELECT 1 FROM sqlite_master WHERE type='table' AND name=?);`
+	for range ticker.C {
+		s.lock.Lock()
+		for _, table := range []string{"jobsets", "jobservice"} {
+			var exists bool
+			err := s.db.QueryRow(tableExistsStmt, table).Scan(&exists)
+			if err != nil {
+				log.Errorf("Error checking table '%s' existence: %v", table, err)
+				continue
+			}
+			if exists {
+				var stmt string
+				if table == "jobsets" {
+					stmt = jobsetsStmt
+				} else {
+					stmt = jobServiceStmt
+				}
+				_, execErr := s.db.Exec(stmt)
```

Comment:
The first returned value (ignored here) is a `database/sql.Result` value. If there's no error here, maybe it might be useful to use the returned `Result` and call `Result.RowsAffected()`, and then maybe log (maybe at DEBUG level?) the number of rows that it deleted? That might be useful for debugging when they run this in a big production setup. What do you think?

Reply:
I agree, it would be good to log it on debug level.